File: ajay/factorial.c

#include <stdio.h>
#include<math.h>

int main()
{
    int a,b=1,sum=1;
    printf("enter the number:");
    scanf("%d",&a);
    if(a<0)
    {
        printf("the factorial for negtive no. doesnt exist");
    }
    else if(a==0)
    {
        printf("the factorial for 0 is 1");
    }
    else
    {
        
        while(b<=a)
    {
        sum=sum*b;
        b=b+1;
    }
    printf("result=%d",sum);
    }
    
    
    return 0;
}

File: Raghul/python/factorial.py
a = int(input("Enter the value to be factorial : "))
factorial = 1
if a < 0:
    print("There is no factorial for ",a)
elif a == 0:
    print("The factorial value for", a ,"is 1")
else :
    for i in range(1 , a + 1):
        factorial = factorial * i
        print(factorial)
    print("The factorial for ",a,"is",factorial)    
